Classes/EventStore/StreamNameAndVersionFilter.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace Neos\EventSourcing\EventStore;
+
+/*
+ * This file is part of the Neos.EventSourcing package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\EventSourcing\Exception;
+
+/**
+ * Stream name and version filter
+ *
+ * This matches a single event in a stream
+ */
+class StreamNameAndVersionFilter implements EventStreamFilterInterface
```

Comment:
This could be a valuable addition, but can be used with the existing `get(EventStreamFilter)` API already.

Reply:
How? I want to filter for a stream *and* the event version.. Alternatively we could filter by the event identifier (and I'll push a PR for that to make progress with this) but it's currently not possible to target a single event

Reply:
I actually meant to add this StreamNameAndVersionFilter, and just use `get($StreamNameAndVersionFilter)->first()` instead of `getOne($StreamNameAndVersionFilter)` - keep the API slim.

Reply:
But what if I don't want to fetch the *first* event of a stream but one with version != 0?

Reply:
`minVersion` filter ;)